spring-boot-admin-server-ui/src/main/java/de/codecentric/boot/admin/server/ui/web/UiController.java

```diff
@@ -118,6 +118,7 @@ public String login() {
         private final String favicon;
         private final String faviconDanger;
         private final boolean notificationFilterEnabled;
+        private final boolean rememberMeEnabled;
```

Comment:
Please add a corresponding property to the `AdminServerUiProperties` which is mapped to the `UiSettings` in `AdminServerUiAutoConfiguration#homeUiController()`. Also the docs for this new config prop needs mention in the docs.